src/index.html

```diff
@@ -11,7 +11,222 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="POST"
+    onsubmit="onSubmit()"
+    >
```

Comment:
+2 spaces for attributes.
```suggestion
    <form
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      method="POST"
      onsubmit="onSubmit()"
    >
```